plugins/dell/fu-plugin-dell.c

```diff
@@ -8,129 +8,138 @@
 
 #include "config.h"
 
-#include <fwup.h>
 #include <appstream-glib.h>
+#include <fcntl.h>
+#include <fwup.h>
 #include <glib/gstdio.h>
 #include <smbios_c/system_info.h>
 #include <string.h>
 #include <unistd.h>
-#include <fcntl.h>
 
+#include "fu-device-metadata.h"
 #include "fu-plugin-dell.h"
 #include "fu-plugin-vfuncs.h"
-#include "fu-device-metadata.h"
 
 /* These are used to indicate the status of a previous DELL flash */
-#define DELL_SUCCESS			0x0000
-#define DELL_CONSISTENCY_FAIL		0x0001
-#define DELL_FLASH_MEMORY_FAIL		0x0002
-#define DELL_FLASH_NOT_READY		0x0003
-#define DELL_FLASH_DISABLED		0x0004
-#define DELL_BATTERY_MISSING		0x0005
-#define DELL_BATTERY_DEAD		0x0006
-#define DELL_AC_MISSING			0x0007
-#define DELL_CANT_SET_12V		0x0008
-#define DELL_CANT_UNSET_12V		0x0009
-#define DELL_FAILURE_BLOCK_ERASE	0x000A
-#define DELL_GENERAL_FAILURE		0x000B
-#define DELL_DATA_MISCOMPARE		0x000C
-#define DELL_IMAGE_MISSING		0x000D
-#define DELL_DID_NOTHING		0xFFFF
+#define DELL_SUCCESS 0x0000
+#define DELL_CONSISTENCY_FAIL 0x0001
```

Comment:
NAK, this is pretty wrong.

Reply:
Yeah until the one I linked in the top comment is merged into clang this will need to be clang-format guarded.